### PR TITLE
Update kubeflow sdk version

### PR DESCRIPTION
**Pull request by jomach**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/688)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
@jomach Have you signed the CLA? See the previous comment from the googlebot.

**Comment by jomach** [comment]
@sarahmaddox Hello, Yes but only after creating the PR. Do I need to re-create the pr ? Sorry first time on this project

**Comment by jomach** [comment]
cool, a new comment is enough.

**Comment by jinchihe** [comment]
@jomach To avoid similar problems, in fact, personally I suggest to reminder user to download latest version of the sdk (need to describe how to get latest one). Otherwise, need to update this when the new pkg release after some days. Thanks.

**Comment by jomach** [comment]
@jinchihe So the sdk version does not match the version of kubeflow itself. I did the trick of just searching for the latest version on  https://storage.googleapis.com/ml-pipeline/ which I cannot recommend for normal users.  Where would be a good place to point users to the correct version ?

**Comment by elikatsis** [comment]
@jomach 
>Where would be a good place to point users to the correct version ?

Probably this page: https://github.com/kubeflow/pipelines/releases

**Comment by jomach** [comment]
I updated the docs. Please check

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-688--competent-brattain-de2d6d.netlify.com/docs/pipelines/sdk/install-sdk/

File: content/docs/pipelines/sdk/install-sdk.md [hunk 1]
```diff
@@ -71,7 +71,8 @@ up Python using [Miniconda](https://conda.io/miniconda.html):
 Run the following command to install the Kubeflow Pipelines SDK:
 
 ```bash
-pip install https://storage.googleapis.com/ml-pipeline/release/{{% pipelines-sdk-version %}}/kfp.tar.gz --upgrade
+latest_version=$(curl --silent https://api.github.com/repos/kubeflow/pipelines/releases/latest | jq -r .tag_name)
+pip install https://storage.googleapis.com/ml-pipeline/release/$latest_version/kfp.tar.gz --upgrade
```
Comment (sarahmaddox) [inline content/docs/pipelines/sdk/install-sdk.md]: I think the variable should be enclosed in curly brackets:

```
pip install https://storage.googleapis.com/ml-pipeline/release/${latest_version}/kfp.tar.gz --upgrade
```

Comment (jomach) [reply content/docs/pipelines/sdk/install-sdk.md]: updated. 

One question. Where is the code where it performs one deployment per branch ? Is it Jenkins ?

Comment (sarahmaddox) [reply content/docs/pipelines/sdk/install-sdk.md]: It's Netlify that builds the doc deployment. To find the preview for your PR:

* Wait for the automated PR workflow to do some checks. When it's ready,
  you should see a comment like this: **deploy/netlify — Deploy preview ready!**
* Click **Details** to the right of "Deploy preview ready" to see a preview
  of your updates.

**Comment by sarahmaddox** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**